vcpkg.json

```diff
@@ -22,11 +22,53 @@
       "description": "Build with Qt 6",
       "dependencies": [
         "qt5compat",
-        "qtbase",
+        {
+          "name": "qtbase",
+          "default-features": false,
+          "features": [
+            "brotli",
+            "concurrent",
+            {
+              "name": "dbus",
+              "platform": "!emscripten"
+            },
+            "default-features",
+            "doubleconversion",
+            "freetype",
+            "gui",
+            "harfbuzz",
+            "icu",
+            "jpeg",
+            "network",
+            "opengl",
+            {
+              "name": "openssl",
+              "platform": "!emscripten"
+            },
+            "pcre2",
+            "png",
+            "sql",
+            {
+              "name": "sql-psql",
+              "platform": "!emscripten"
+            },
+            "sql-sqlite",
+            "testlib",
+            "thread",
+            "widgets",
+            "zstd"
+          ]
+        },
```

Comment:
As with iOS (#117) we have to list the Qt features explicitly to exclude features for Emscripten as needed. https://github.com/mixxxdj/vcpkg/pull/117#issuecomment-1930635110 applies here too, once we update Qt, we should include the `gles3` feature here too (for `emscripten`).